README.md

```diff
@@ -1,2 +1,27 @@
-# pytest-browserstack
-https://browserstack.atlassian.net/wiki/spaces/PROD/pages/3358687296/Selenium+Frameworks+Documentation#Git-Repo-Names
+# PyTest with Browserstack
+
+PyTest Integration with BrowserStack.
+
+![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
+
+## Setup
+
+* Clone the repo
+* Install dependencies `pip install -r requirements.txt`
+* To run your automated tests using BrowserStack, you must provide a valid username and access key. This can be done either by using a .browserstack configuration file in the working directory or your home directory, or by setting the BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY environment variables.
+
+## Running your single tests
+* To run single tests, run `paver run single`
```

Comment:
Don't need single sample test, having || and local will suffice. Single test file and config file can be removed as well.

Reply:
As per [guidelines](https://browserstack.atlassian.net/wiki/spaces/PROD/pages/3358687296/Selenium+Frameworks+Documentation#BrowserStack-Git-Repo) having single,parallel & local files